util/ranger/points.go

```diff
@@ -284,19 +284,24 @@ func (r *builder) buildFromIsFalse(expr *expression.ScalarFunction, isNot int) [
 	return []point{startPoint, endPoint}
 }
 
-func (r *builder) newBuildFromIn(expr *expression.ScalarFunction) []point {
-	var rangePoints []point
+func (r *builder) newBuildFromIn(expr *expression.ScalarFunction) ([]point, bool) {
```

Comment:
Why is it named `newBuildFromIn` instead of `BuildFromIn` ?

Reply:
Maybe there is a time that there're two `buildFromIn` method...